src/python/dxpy/scripts/dx.py

```diff
@@ -858,7 +858,7 @@ def rm(args):
             try:
                 with nostderr():
                     project, folderpath, entity_results = resolve_existing_path(path, allow_mult=True, all_mult=args.all)
-                if folderpath == '/':
+                if folderpath == '/' and entity_results is None:
                     print("")
```

Comment:
does this additional check `entity_results = None` mean that the user is describing the root folder of the project, and nothing else?

Reply:
Yes, if `entity_results` is something other than `None` then there is some object being described.